admin_manual/configuration/server/security/ransomeware-protection.rst

```diff
@@ -0,0 +1,95 @@
+It aims to do so in two ways: `prevention <ransomware_prevention_label>`_, and `protection <ransomware_protection_label>`_.
+
+.. _ransomware_prevention_label:
+
+Prevention: Blocking Common Ransomware File Extensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Like other forms of cyberattack, Ransomware has a range of diverse characteristics.
+On the one hand it makes them hard to detect and on the other makes them even harder to prevent.
+Recent ransomware attacks either encrypts a user's files and adds a certain file extension (e.g., ".crypt") or replaces the original files with an encrypted copy and a certain file extension.
+
+The first line of defense against such threats is a blacklist that blocks uploading files with file extensions known to originate from ransomware.
+Ransomware Protection ships with `a static extension list`_ of around 1,500 file extensions.
+As new extensions are regularly created this list needs maintenance.
+Future releases of Ransomware Protection will include an updated list and the ability to update the list via syncing with `FSRM's API`_ by using :doc:`occ <../occ_command>`.
+
+.. _ransomware_protection_label:
+
+Protection: Data Retention and Rollback
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While Ransomware Prevention mitigates risks of a range of ransomware attacks, it is not a future-proof solution, because ransomware is becoming ever-more sophisticated.
+There are known attacks that change file extensions randomly or keep them unchanged which makes them harder to detect.
+
+Ultimately there is a consensus that only one solution can provide future-proof protection from ransomware attacks: retaining data and providing the means to roll back to a particular point in time.
+
+ownCloud Ransomware Protection will, therefore, record all changes on an ownCloud Server and allow administrators to rollback user data to a particular point in time, making use of ownCloud’s integrated Versioning and Trash bin features.
+
+Doing so allows all user data that is synchronized with the server to be rolled back to its state before the attack occurred.
+We expect that a combination of Ransomware prevention and protection should reduce risks to a minimum acceptable level.
+
+Other Elements of Ransomware Protection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+==================== ============================================== ================================================
+Name                 Command                                        Description
+==================== ============================================== ================================================
+Ransomguard Scanner  ``occ ransomguard:scan <timestamp> <user>``    A command to scan the ownCloud database for
+                                                                    anomalies in a user account. It enables an administrator to determine the point in time
```

Comment:
Maybe s/anomalies/changes/ ?

Reply:
"A command to scan the ownCloud database for changes in order to discover anomalies and their origin" ?

Reply:
Commented in conversation.